Sources/SWBCore/Settings/Settings.swift

```diff
@@ -2173,13 +2173,22 @@ private class SettingsBuilder {
         }
         do {
             let scope = createScope(sdkToUse: sdk)
+            var table = MacroValueAssignmentTable(namespace: core.specRegistry.internalMacroNamespace)
 
             // If the product is being built as mergeable, then that overrides certain other settings.
             if scope.evaluate(BuiltinMacros.MAKE_MERGEABLE) {
-                var table = MacroValueAssignmentTable(namespace: core.specRegistry.internalMacroNamespace)
                 table.push(BuiltinMacros.STRIP_INSTALLED_PRODUCT, literal: false)
-                push(table, .exportedForNative)
             }
+
+            // Even if not being merged in this build, a mergeable library still uses a generated bundle lookup helper to power #bundle support.
+            if scope.evaluate(BuiltinMacros.MERGEABLE_LIBRARY) {
```

Comment:
I wonder if this effectively evaluates to true in a debug build when we're not building as mergeable? cc @mhrawdon